topics/games.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html>
+	<body>
+		<h1>Methods 2022 - Topics - Favorite Video/Board/Other Game<\h1>
```

Comment:
![image](https://user-images.githubusercontent.com/47065429/154158024-315c8df4-01fd-4e9c-8006-daab8b047652.png)
You have some incorrect syntax here, perhaps replace '\' with '/'